client_it_test.go

```diff
@@ -140,6 +143,64 @@ func TestClientMemberEvents(t *testing.T) {
 	})
 }
 
+func calcPartitionID(ss *serialization.Service, key interface{}) (int32, error) {
```

Comment:
I think this function may be named better. Maybe `calculatePartitionID` as the last resort.

Reply:
Sadly, could not come up with a better one. Renamed at https://github.com/hazelcast/hazelcast-go-client/pull/696/commits/5fadb182c34b30145d6eb696303670f96354140f